tests/wpt/web-platform-tests/common/canvas-tests.js

```diff
@@ -90,6 +90,7 @@ function addCrossOriginYellowImage()
     var img = new Image();
     img.id = "yellow.png";
     img.className = "resource";
-    img.src = get_host_info().HTTP_REMOTE_ORIGIN + "/images/yellow.png";
+    img.src = get_host_info().HTTP_ORIGIN + "/common/redirect.py?location=" +
+        get_host_info().HTTP_REMOTE_ORIGIN + "/images/yellow.png";
```

Comment:
Can we add a comment explaining why we're using a redirect? Should we test cross-origin images as well as same-origin redirects to cross-origin images?